packages/twenty-front/src/modules/settings/data-model/components/SettingsObjectFieldSelectForm.tsx

```diff
@@ -169,18 +175,16 @@ export const SettingsDataModelFieldSelectForm = ({
                       itemComponent={
                         <SettingsObjectFieldSelectFormOptionRow
                           key={option.value}
-                          isDefault={option.isDefault}
                           onChange={(nextOption) => {
-                            const nextOptions =
-                              isMultiSelect || !nextOption.isDefault
-                                ? [...options]
-                                : // Reset simple Select default option before setting the new one
-                                  options.map<SettingsObjectFieldSelectFormOption>(
-                                    (value) => ({ ...value, isDefault: false }),
-                                  );
+                            const nextOptions = [...options];
+
```

Comment:
I think toSpliced() can be used if you don't want the need to make a copy yourself (nextOptions)